code/modules/mob/living/simple_mob/subtypes/humanoid/mercs/mercs.dm

```diff
@@ -258,9 +258,80 @@
 
 	corpse = /obj/effect/landmark/mobcorpse/syndicatecommando
 
+	loot_list = list(/obj/item/gun/projectile/automatic/c20r = 100)
+	base_attack_cooldown = 5 // Two attacks a second or so.
+	reload_max = 20
+
 /mob/living/simple_mob/humanoid/merc/ranged/space/Process_Spacemove(var/check_drift = 0)
 	return
 
+// suppressors are just assholes and are intended to be a piss poor experience for everyone on both sides
+
+/datum/ai_holder/simple_mob/merc/ranged/suppressor
+	respect_alpha = FALSE // he really just shoots you
+	vision_range = 10 // plutonia experience
+
+/mob/living/simple_mob/humanoid/merc/ranged/space/suppressor // adminspawn only, and also Probably Going To Kill The Unprepared
+	name = "syndicate suppressor"
```

Comment:
Change name to "Mercenary Suppressor".
Syndicate is dead as a faction in RP lore.